calico/getting-started/kubernetes/self-managed-public-cloud/gce.mdx

```diff
@@ -67,33 +67,33 @@ gcloud compute firewall-rules create example-k8s-allow-external \
   --source-ranges 0.0.0.0/0
 ```
 
-Create the controller VM:
+5. Create the controller VM.
 
 ```
 gcloud compute instances create controller \
     --async \
     --boot-disk-size 200GB \
     --can-ip-forward \
-    --image-family ubuntu-1804-lts \
-    --image-project ubuntu-os-cloud \
+    --image-family ubuntu-pro-1604-lts \
```

Comment:
Why change this to 1604? Isn't that EOL?

Reply:
It may be EOL but it is working. The original image name was wrong. So, I changed it to a working image name. Here's a list of images available on GCP. The one I added is one of them. Perhaps I can change it to `ubuntu-pro-1804-lts`, which is closer to the original one.

```
ubuntu-pro-1604-xenial-v20221201                      ubuntu-os-pro-cloud  ubuntu-pro-1604-lts                            READY
ubuntu-pro-1804-bionic-arm64-v20230522                ubuntu-os-pro-cloud  ubuntu-pro-1804-lts-arm64                      READY
ubuntu-pro-1804-bionic-v20230522                      ubuntu-os-pro-cloud  ubuntu-pro-1804-lts                            READY
ubuntu-pro-2004-focal-arm64-v20230629                 ubuntu-os-pro-cloud  ubuntu-pro-2004-lts-arm64                      READY
ubuntu-pro-2004-focal-v20230629                       ubuntu-os-pro-cloud  ubuntu-pro-2004-lts                            READY
ubuntu-pro-2204-jammy-arm64-v20230630                 ubuntu-os-pro-cloud  ubuntu-pro-2204-lts-arm64                      READY
ubuntu-pro-2204-jammy-v20230630                       ubuntu-os-pro-cloud  ubuntu-pro-2204-lts                            READY
ubuntu-pro-fips-1804-bionic-v20230530                 ubuntu-os-pro-cloud  ubuntu-pro-fips-1804-lts                       READY
ubuntu-pro-fips-2004-focal-v20230614                  ubuntu-os-pro-cloud  ubuntu-pro-fips-2004-lts                       READY
ubuntu-2004-focal-arm64-v20230628                     ubuntu-os-cloud      ubuntu-2004-lts-arm64                          READY
ubuntu-2004-focal-v20230628                           ubuntu-os-cloud      ubuntu-2004-lts                                READY
ubuntu-2204-jammy-arm64-v20230630                     ubuntu-os-cloud      ubuntu-2204-lts-arm64                          READY
ubuntu-2204-jammy-v20230630                           ubuntu-os-cloud      ubuntu-2204-lts                                READY
ubuntu-2210-kinetic-amd64-v20230615                   ubuntu-os-cloud      ubuntu-2210-amd64                              READY
ubuntu-2210-kinetic-arm64-v20230615                   ubuntu-os-cloud      ubuntu-2210-arm64                              READY
ubuntu-2304-lunar-amd64-v20230621                     ubuntu-os-cloud      ubuntu-2304-amd64                              READY
ubuntu-2304-lunar-arm64-v20230621                     ubuntu-os-cloud      ubuntu-2304-arm64                              READY
ubuntu-minimal-2004-focal-arm64-v20230630             ubuntu-os-cloud      ubuntu-minimal-2004-lts-arm64                  READY
ubuntu-minimal-2004-focal-v20230630                   ubuntu-os-cloud      ubuntu-minimal-2004-lts                        READY
ubuntu-minimal-2204-jammy-arm64-v20230629             ubuntu-os-cloud      ubuntu-minimal-2204-lts-arm64                  READY
ubuntu-minimal-2204-jammy-v20230629                   ubuntu-os-cloud      ubuntu-minimal-2204-lts                        READY
ubuntu-minimal-2210-kinetic-amd64-v20230623           ubuntu-os-cloud      ubuntu-minimal-2210-amd64                      READY
ubuntu-minimal-2210-kinetic-arm64-v20230623           ubuntu-os-cloud      ubuntu-minimal-2210-arm64                      READY
ubuntu-minimal-2304-lunar-amd64-v20230621             ubuntu-os-cloud      ubuntu-minimal-2304-amd64                      READY
ubuntu-minimal-2304-lunar-arm64-v20230621             ubuntu-os-cloud      ubuntu-minimal-2304-arm64                      READY
```

Reply:
Unless there are insurmountable obstacles, we should make sure to be using a supported version of Ubuntu. That is, both supported by Ubuntu AND supported by Calico. 1604 is neither.

I get that you're just updating a name that was wrong. But we may as well provide a good image name to future-proof this doc.

I'd say let's go to the latest LTS version that we support.

Reply:
Ubuntu 16.04 seems to be supported by Calico.
https://docs.tigera.io/calico/next/getting-started/kubernetes/requirements

Reply:
@girishshankaran Thanks. That mention of 1604 is almost certainly wrong. These requirements need to be updated.

Reply:
Sure, will create a ticket for that.